components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -233,7 +233,7 @@ In order to find a token's ID, visit the *API Tokens* page.
 
 ### Projects
 
-Projects are used to group and permission Chef Automate resources as well as ingested data, specifically Compliance reports, Chef Server and Compliance events, and Infrastructure nodes.
+Projects are used to group and permission Chef Automate resources as well as ingested data, specifically Compliance reports, Chef Server events, and Infrastructure nodes.
```

Comment:
Removing the "Compliance events" part because is not implemented in Automate. Meaning that we are not tagging or filtering on projects for compliance events.